transport/http/outbound.go

```diff
@@ -111,6 +111,7 @@ func (t *Transport) NewOutbound(chooser peer.Chooser, opts ...OutboundOption) *O
 	for _, opt := range opts {
 		opt(o)
 	}
+	o.sender = &transportSender{Client: t.client}
```

Comment:
t.client is either provided by opts or from [default option](https://github.com/yarpc/yarpc-go/blob/dev/transport/http/transport.go#L66). It always exists.